Library/Homebrew/extend/ARGV.rb

```diff
@@ -268,6 +268,10 @@ def fetch_head?
     include? "--fetch-HEAD"
   end
 
+  def ignore_installed_specs?
+    include? "--ignore-installed-specs"
```

Comment:
> we should warn and ask user to specify some option:

I don't think we should. 2. is a bad option. I think we should always do 3. unless `--build-from-source` is passed. I may well be missing something here.